modules/ppcp-api-client/src/Entity/ExperienceContext.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * The experience_context object.
+ *
+ * @package WooCommerce\PayPalCommerce\ApiClient\Entity
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\ApiClient\Entity;
+
+use ReflectionClass;
+
+/**
+ * Class ExperienceContext
+ */
+class ExperienceContext {
+
+	const LANDING_PAGE_LOGIN          = 'LOGIN';
+	const LANDING_PAGE_GUEST_CHECKOUT = 'GUEST_CHECKOUT';
+	const LANDING_PAGE_NO_PREFERENCE  = 'NO_PREFERENCE';
+
+	const SHIPPING_PREFERENCE_GET_FROM_FILE        = 'GET_FROM_FILE';
+	const SHIPPING_PREFERENCE_NO_SHIPPING          = 'NO_SHIPPING';
+	const SHIPPING_PREFERENCE_SET_PROVIDED_ADDRESS = 'SET_PROVIDED_ADDRESS';
+
+	const USER_ACTION_CONTINUE = 'CONTINUE';
+	const USER_ACTION_PAY_NOW  = 'PAY_NOW';
+
+	const PAYMENT_METHOD_UNRESTRICTED               = 'UNRESTRICTED';
+	const PAYMENT_METHOD_IMMEDIATE_PAYMENT_REQUIRED = 'IMMEDIATE_PAYMENT_REQUIRED';
```

Comment:
Do we need to set the visibilities for these?

Reply:
Looks like const properties are public by default, private const is allowed since PHP 7.1.

Reply:
Yes, even though it's not required, I think explicitly setting visibility for clarity and consistency is better (especially when private const or protected const are also already used). But not a strong opinion, so please feel free to ignore.